core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -491,6 +484,14 @@ object H2OContext extends Logging {
     val sameNodes = hc.getH2ONodes().map(_.ipPort()).sameElements(newCloudV3.nodes.map(_.ip_port))
     !sameNodes
   }
+
+  private def checkConf(conf: H2OConf): H2OConf = {
```

Comment:
nit: Shouldn't the method name correspond to methods that are called. `checkAndUpdateConf`

Reply:
Yup, like the proposal, thank you!